azkaban-common/src/test/java/azkaban/executor/InteractiveTestJob.java

```diff
@@ -141,4 +142,10 @@ public void cancel() throws InterruptedException {
     info("Killing job");
     failJob();
   }
+
+  public static void clearTestJobs(final String... names) {
```

Comment:
So only one thread will call this method at any given time?
Can it ever happen that one thread starts to remove testJobs while another assumes that testJobs are there?